openassessment/xblock/openassessmentblock.py

```diff
@@ -834,7 +834,11 @@ def is_released(self, step=None):
         else:
             is_published = True
         is_closed, reason, __, __ = self.is_closed(step=step)
-        return is_published and (not is_closed or reason == 'due')
+        is_released = is_published and (not is_closed or reason == 'due')
+        is_live = dt.datetime.now(pytz.UTC) > parse_date_value(self.start, self._) if self.start else None
```

Comment:
I like this, but I think it would read a bit cleaner if we're more explicit about checking `self.start`, and avoid having `is_live` be None.

Perhaps something like this:
```
if self.start:
    is_live = ...
    is_released = is_released and is_live
```